src/encoded/static/components/browse/FileSearchView.js

```diff
@@ -5,59 +5,38 @@ import PropTypes from 'prop-types';
+import { DisplayTitleColumnWrapper, DisplayTitleColumnDefault } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/table-commons';
 import { console } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
 import { SelectedFilesController } from './../browse/components/SelectedFilesController';
-import { navigate, Schemas } from './../util';
-import { FacetList } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/FacetList';
+import { navigate } from './../util';
 import { columnExtensionMap as colExtensionMap4DN } from './columnExtensionMap';
-import { WindowNavigationController } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/WindowNavigationController';
 import { transformedFacets } from './SearchView';
 import { AboveBrowseViewTableControls } from './components/above-table-controls/AboveBrowseViewTableControls';
 import { fileToAccessionTriple } from './../util/experiments-transforms';
-import { SearchResultTable } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/SearchResultTable';
 
-export default class FileSearchView extends React.PureComponent {
 
-    constructor(props) {
-        super(props);
-
-        this.memoized = {
-            transformedFacets: memoize(transformedFacets)
-        };
-
-        this.state = {
-            show: false,
-            confirmLoading: false
-        };
-    }
-
-    render() {
-        const { isFullscreen, href, context, currentAction, session, schemas } = this.props;
-        const facets = this.memoized.transformedFacets(href, context, currentAction, session, schemas);
-
-        return (
-            <div className="search-page-container container" id="content">
-                <SelectedFilesController {...{ href, context }}>
-                    <FileTableWithSelectedFilesCheckboxes {...this.props} facets={facets} />
-                </SelectedFilesController>
-            </div>
-        );
-    }
+export default function FileSearchView (props){
+    const { href, context } = props;
+    return (
+        <div className="search-page-container container" id="content">
+            <SelectedFilesController {...{ href, context }}>
+                <FileTableWithSelectedFilesCheckboxes {...props} />
+            </SelectedFilesController>
+        </div>
+    );
 }
 
+
 function FileTableWithSelectedFilesCheckboxes(props){
     const {
         // Common high-level props from Redux, or App.js, or App.js > BodyElement:
         context, href, schemas, navigate: propNavigate,
         windowHeight, windowWidth, registerWindowOnScrollHandler,
         toggleFullScreen, isFullscreen, session,
-        facets,
-        selectedFiles, selectFile, unselectFile, resetSelectedFiles,
-        columnExtensionMap
+        selectedFiles, selectFile, unselectFile, resetSelectedFiles, selectedFilesUniqueCount,
+        columnExtensionMap,
+        currentAction
```

Comment:
Codacy found an issue: ['currentAction' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6342004)